Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -692,6 +692,58 @@ of issues:
   }
 }
 
+### Run tests sequentially
+
+By default, the testing library runs all tests in a suite in parallel. The
+default behavior of XCTest is to run each test in a suite sequentially. If your tests use
+shared state, for example, global variables, you may see expected behavior
+including unreliable test outcomes when you run tests in parallel.
+
+Annotate your test suite with ``Trait/serialized`` to run tests within that
+suite serially:
+
+@Row {
+  @Column {
+    ```swift
+    // Before
+    class RefrigeratorTests : XCTestCase
+      func testLightComesOn() async {
+        try FoodTruck.shared.refrigerator.openDoor()
+        XCTAssertEqual(FoodTruck.shared.refrigerator.lightState == .on)
+      }
+      
+      func testLightGoesOut() async {
+        try FoodTruck.shared.refrigerator.openDoor()
+        try FoodTruck.shared.refrigerator.closeDoor()
+        XCTAssertEqual(FoodTruck.shared.refrigerator.lightState == .off)
+      }
+    }
+    ```
+  }
+  @Column {
+    ```swift
+    // After
+    @Suite(.serialized)
+    class RefrigeratorTests {
+      @Test
```

Comment:
Please put `@Test` and `func` on the same line (it makes the side-by-side view clearer.)